packages/providers/src.ts/fallback-provider.ts

```diff
@@ -289,26 +289,27 @@ function getProcessFunc(provider: FallbackProvider, method: string, params: { [
             break;
 
         // We drop the confirmations from transactions as it is approximate
-        case "getBlock":
-            // We drop the confirmations from transactions as it is approximate
-            if (params.includeTransactions) {
-                normalize = function(block: BlockWithTransactions): string {
-                    if (block == null) { return null; }
-
-                    block = shallowCopy(block);
-                    block.transactions = block.transactions.map((tx) => {
-                        tx = shallowCopy(tx);
-                        return tx;
-                    });
-                    return serialize(block);
-                };
-            } else {
-                normalize = function(block: Block): string {
-                    if (block == null) { return null; }
-                    return serialize(block);
-                }
-            }
-            break;
+        // case "getBlock":
+        //     // We drop the confirmations from transactions as it is approximate
+        //     if (params.includeTransactions) {
+        //         normalize = function(block: BlockWithTransactions): string {
+        //             if (block == null) { return null; }
+
+        //             block = shallowCopy(block);
+        //             block.transactions = block.transactions.map((tx) => {
+        //                 tx = shallowCopy(tx);
+        //                 tx.confirmations = -1;
+        //                 return tx;
+        //             });
+        //             return serialize(block);
+        //         };
+        //     } else {
+        //         normalize = function(block: Block): string {
+        //             if (block == null) { return null; }
+        //             return serialize(block);
+        //         }
+        //     }
+        //     break;
```

Comment:
maybe we should remove this altogether as hedera has no concept for blocks

Reply:
I found the cause of this change and will revert it.